building/configlet/lint.md

```diff
@@ -70,6 +70,29 @@ The `config.json` file should have the following checks:
 - The `"online_editor.indent_size"` value must be an integer >= 0 and <= 8
 - The `"online_editor.highlightjs_language"` key is required
 - The `"online_editor.highlightjs_language"` value must be a non-blank string¹
+- The `"files"` key is optional
+- The `"files"` value must be an object
+- The `"files.solution`" key is optional
+- The `"files.solution`" value must be an array
+- The `"files.solution`" values must be valid patterns⁴
+- The `"files.solution`" values must not have duplicates
+- The `"files.test`" key is optional
+- The `"files.test`" value must be an array
+- The `"files.test`" values must be valid patterns⁴
+- The `"files.test`" values must not have duplicates
+- The `"files.example`" key is optional
+- The `"files.example`" value must be an array
+- The `"files.example`" values must be valid patterns⁴
+- The `"files.example`" values must not have duplicates
+- The `"files.exemplar`" key is optional
+- The `"files.exemplar`" value must be an array
+- The `"files.exemplar`" values must be valid patterns⁴
+- The `"files.exemplar`" values must not have duplicates
+- The `"files.editor`" key is optional
+- The `"files.editor`" value must be an array
```

Comment:
Would it be clear that while the key is optional, _if_ it is present, it must be a non-empty array?

```suggestion
- The `"files.editor`" value must be a non-empty array
```